packages/headless/doc-parser/src/actions-resolver.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  ApiEntryPoint,
+  ApiItemKind,
+  ApiVariable,
+} from '@microsoft/api-extractor-model';
+import {DocNode, DocParamBlock} from '@microsoft/tsdoc';
+import {AnyEntity, Entity} from './entity';
+import {emitAsTsDoc} from './tsdoc-emitter';
+
+interface ActionsNamespace {
+  name: string;
+  members: Action[];
+}
+
+enum ActionKind {
+  Action = 'action',
+  AsyncAction = 'asyncThunk',
+  AnalyticsAction = 'analyticsAction',
+}
+
+interface Action {
+  desc: string;
+  fullName: string;
+  parameter: AnyEntity | null;
+  actionKind: ActionKind;
+}
+
+export function resolveActionNamespaces(
+  entry: ApiEntryPoint
+): ActionsNamespace[] {
+  return entry.members
+    .filter(
+      (member) =>
+        member.kind === ApiItemKind.Namespace &&
+        member.displayName.endsWith('Actions')
```

Comment:
Namespaces are currently identified as those suffixed by `Actions`, but this won't be necessary if a configuration similar to controllers' is used to define them individually.

Otherwise, alternatives would be to identify namespaces by their file path or by a comment tag (which I would rather not use, since the api extractor doesn't have great support for custom tags).